lms/djangoapps/courseware/grades.py

```diff
@@ -162,7 +162,14 @@ def score_for_module(self, location):
         if location in self.weighted_scores:
             score = self.weighted_scores[location]
             return score.earned, score.possible
-        children = self.locations_to_children[location]
+        children = None
+        if location_clear:
+            for loc, dt in self.locations_to_children.iteritems():
+                if loc.replace(version=None, branch=None) == location:
+                    children = dt
+                    break
+        if children is None:
+            children = self.locations_to_children[location]
```

Comment:
Why not strip the version/branch info when creating the ProgressSummary [here](https://github.com/StraNNiKK/edx-platform/blob/master/lms/djangoapps/courseware/grades.py#L616)?